ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -63,6 +63,9 @@ interface {{ ethernet_interface }}
    no switchport
 {%         elif ethernet_interfaces[ethernet_interface].type is arista.avd.defined('l3dot1q') and ethernet_interfaces[ethernet_interface].encapsulation_dot1q_vlan is arista.avd.defined() %}
    encapsulation dot1q vlan {{ ethernet_interfaces[ethernet_interface].encapsulation_dot1q_vlan }}
+{%         elif ethernet_interfaces[ethernet_interface].type is arista.avd.defined('l2dot1q') and ethernet_interfaces[ethernet_interface].encapsulation_vlan.client_dot1q is arista.avd.defined() %}
```

Comment:
missing documentation, molecule and readme update for this change to `eos_cli_config_gen`

Reply:
Fixed the data model and added README info + updated doc templates.